solid-acp.ttl

```diff
@@ -0,0 +1,273 @@
+
+
+####################
+#   Properties
+####################
+acp:resource
+    a rdf:Property ;
+    owl:inverseOf acp:accessControlResource ;
+    rdfs:label "resource"@en ;
+    rdfs:comment "The resource property connects ACRs to resources they control. It is the inverse of acp:accessControlResource."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControlResource .
+
+acp:accessControlResource
+    a rdf:Property ;
+    owl:inverseOf acp:resource ;
+    rdfs:label "access control resource"@en ;
+    rdfs:comment "The access control resource property connects resources to ACRs controlling access to them. It is the inverse of acp:resource."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:range acp:AccessControlResource .
+
+acp:accessControl
+    a rdf:Property ;
+    rdfs:label "access control"@en ;
+    rdfs:comment "The access control property connects ACRs to access controls."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControlResource ;
+    rdfs:range acp:AccessControl .
+
+acp:memberAccessControl
+    rdfs:label "member access control"@en ;
+    rdfs:comment "The member access control property connects ACRs of member resources to access controls."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControlResource ;
+    rdfs:range acp:AccessControl .
+
+acp:apply
+    a rdf:Property ;
+    rdfs:label "apply"@en ;
+    rdfs:comment "The apply property connects Access Controls to the Policies they apply to resources."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControl ;
+    rdfs:range acp:Policy .
+
+acp:allow
+    a rdf:Property ;
+    rdfs:label "allow"@en ;
+    rdfs:comment "The allow property connects Policies to the Access Modes they allow if satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:AccessMode .
+
+acp:deny
+    a rdf:Property ;
+    rdfs:label "deny"@en ;
+    rdfs:comment "The deny property connects Policies to the Access Modes they deny if satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:AccessMode .
+
+acp:allOf
+    a rdf:Property ;
+    rdfs:label "all of"@en ;
+    rdfs:comment "The all of property connects Policies to the set of Matchers that must all match a resource access description for the policy to be satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:Matcher .
+
+acp:anyOf
+    a rdf:Property ;
+    rdfs:label "any of"@en ;
+    rdfs:comment "The any of property connects Policies to the set of Matchers any of which must match a resource access description for the policy to be satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:Matcher .
+
+acp:noneOf
+    a rdf:Property ;
+    rdfs:label "none of"@en ;
+    rdfs:comment "The none of property connects Policies to the set of Matchers none of which must match a resource access description for the policy to be satisfied."@en ;
```

Comment:
"none of which must match" *might* be better rephrased to "all of which must not match" ... but I think my suggested rephrase below (i.e., "none of which may match") is better.
```suggestion
    rdfs:comment "The noneOf property connects Policies to a set of Matchers, none of which may match a resource access description for the policy to be satisfied."@en ;
```